src/status_im/hardwallet/card.cljs

```diff
@@ -51,12 +51,24 @@
              platform/android?)
     (.addListener event-emitter "keyCardOnDisconnected" callback)))
 
+(defn on-nfc-enabled [callback]
+  (when (and config/hardwallet-enabled?
```

Comment:
why do we need this when in every method ?

Reply:
Let's keep it as is, I don't want to touch stuff unrelated to this PR, we will go fo refactoring some time soon